source/frontend/src/components/common/form/ParentSelect.test.tsx

```diff
@@ -66,8 +65,10 @@ it('changes to corresponding child value on click', async () => {
     });
   });
   const childElement = getByRole('option');
-  await waitFor(() => {
-    fireEvent.click(childElement);
+  await act(async () => {
+    await waitFor(() => {
+      fireEvent.click(childElement);
+    });
```

Comment:
I think the waitfor could be removed here - could you please try if this works?

```tsx
await act(async () => {
    fireEvent.click(childElement);
});
```

Reply:
@buddy326 did you try this?

Reply:
I did and it still works so that change has been made.